backend/tests/conftest.py

```
@@ -0,0 +1,54 @@
+import pytest
+from fastapi import Header
+from pydantic import SecretStr
+
+from api.auth import AuthContext, build_auth_context, get_auth_context, get_current_user
+from core.config import settings
+from main import app
+
+TEST_DEV_AUTH_TOKEN = "test-dev-auth-token"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "TEST_DEV_AUTH_TOKEN"

(S105)